lib/elixir/lib/task.ex

```diff
@@ -12,46 +12,46 @@ defmodule Task do
       res  = do_some_other_work()
       res + Task.await(task)
 
-  Tasks spawned with `async` can be waited on by their caller
+  Tasks spawned with `async` can be awaited on by their caller
   process (and only their caller) as shown in the example above.
   They are implemented by spawning a process that sends a message
   to the caller once the given computation is performed.
 
   Besides `async/1` and `await/2`, tasks can also be
-  started as part of supervision tree and dynamically spawned
-  in remote nodes. We will explore all three scenarios next.
+  started as part of a supervision tree and dynamically spawned
+  on remote nodes. We will explore all three scenarios next.
 
   ## async and await
 
-  One of the common use of tasks is to convert sequential code
+  One of the common uses of tasks is to convert sequential code
   into concurrent code with `Task.async/1` while keeping its semantics.
   When invoked, a new process will be created, linked and monitored
   by the caller. Once the task action finishes, a message will be sent
   to the caller with the result.
 
   `Task.await/2` is used to read the message sent by the task.
-  `await` will check the monitor setup by the call to `async/1` to
+  `await` will check the monitor set up by the call to `async/1` to
   verify if the process exited for any abnormal reason (or in case
   exits are being trapped by the caller).
```

Comment:
I don't think this paragraph needs to say anything else after the first sentence. Talking about checking the monitor implies that the caller will not be forcibly terminated if the task exits, which is not true.

And also this is somewhat weird behaviour: if the caller traps exits, it presumably doesn't want to be killed. But then it calls `Task.await` which calls `exit()` if the task has crashed. The moduledoc already advises against trapping exits. I would add a sentence to it explaining that trapping exits does not prevent the task owner dying when it calls `Task.await`.